manifest_staging/charts/secrets-store-csi-driver/README.md

```diff
@@ -10,14 +10,31 @@ Quick start instructions for the setup and configuration of secrets-store-csi-dr
 
 ### Installing the chart
 
-Add the chart repo
+> Note: The helm chart repository URL has changed from `https://raw.githubusercontent.com/kuberentes-sigs/secrets-store-csi-driver/master/charts` to `https://kuberentes-sigs.github.io/secrets-store-csi-driver/charts`.
```

Comment:
nit: I think it may be fine to remove this note from all the first time install pages and only leave it on the upgrade one, it may 'get in the way' of new users who have to read it and determine whether it matters to them or not.

Reply:
I was thinking the same, but only concern is what if they have the repo added before and plan to install it now. They might reference the doc for the configuration parameters and see the note to change the URL? wdyt?

Reply:
Ah that makes sense, maybe we have a bug for like 3-6 months to just remove this note just so we dont keep it around forever

Reply:
sgtm! Issue: https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues/699